java/parser.js

```diff
@@ -1,171 +1,206 @@
-            return JSON.stringify(selector)
+            return JSON.stringify(selector);
         } else {
             return printSelector(selector);
         }
 
         function checkCss(string) {
-            return (string.includes('[') && string.includes(']')) || string.includes('#')
+            return (string.includes('[') && string.includes(']')) || string.includes('#');
         }
     }
 
     function region(region) {
-        return `.region(${regionParameter(region)})`
+        return `.region(${regionParameter(region)})`;
     }
 
     function ignoreRegions(arr) {
-        return arr.reduce((acc, val) => `${acc}.ignore(${regionParameter(val)})`, '')
+        return arr.reduce((acc, val) => `${acc}.ignore(${regionParameter(val)})`, '');
     }
 
     function layoutRegions(arr) {
-        return arr.reduce((acc, val) => `${acc}.layout(${regionParameter(val)})`, '')
+        return arr.reduce((acc, val) => `${acc}.layout(${regionParameter(val)})`, '');
     }
 
     function floatingRegions(arr) {
-        return arr.reduce((acc, val) => `${acc}.floating(${floating(val)})`, ``)
+        return arr.reduce((acc, val) => `${acc}.floating(${floating(val)})`, ``);
     }
 
     function floating(floating) {
-        let string
-        string = regionParameter(floating.region)
-        string += `, ${floating.maxUpOffset}, ${floating.maxDownOffset}, ${floating.maxLeftOffset}, ${floating.maxRightOffset}`
-        return string
+        let string;
+        string = regionParameter(floating.region);
+        string += `, ${floating.maxUpOffset}, ${floating.maxDownOffset}, ${floating.maxLeftOffset}, ${floating.maxRightOffset}`;
+        return string;
     }
 
     function accessibilityRegions(arr) {
-        return arr.reduce((acc, val) => `${acc}.accessibility(${accessibility(val)})`, ``)
+        return arr.reduce((acc, val) => `${acc}.accessibility(${accessibility(val)})`, ``);
     }
 
     function accessibility(val) {
-        return `${regionParameter(val.region)}, AccessibilityRegionType.${capitalizeFirstLetter(val.type)}`
+        return `${regionParameter(val.region)}, AccessibilityRegionType.${capitalizeFirstLetter(val.type)}`;
     }
 
     function regionParameter(region) {
-        let string
+        let string;
         switch (typeof region) {
             case 'string':
-                string = `By.cssSelector(${JSON.stringify(region)})`
+                string = `By.cssSelector(${JSON.stringify(region)})`;
                 break;
             case "object":
                 if (region.type) {
-                    string = native ? `getDriver().findElement(${parseObject(region)})` : parseObject(region)
+                    string = native ? `getDriver().findElement(${parseObject(region)})` : parseObject(region);
                 } else {
-                    string = parseObject({value: region, type: 'Region'})
+                    string = parseObject({value: region, type: 'Region'});
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
                    string = parseObject({value: region, type: "Region"});
```